cluster/saltbase/salt/kube-scheduler/kube-scheduler.manifest

```diff
@@ -10,7 +10,12 @@
  {% set feature_gates = "--feature-gates=" + grains.feature_gates -%}
 {% endif -%}
 
-{% set params = params + log_level + " " + feature_gates -%}
+{% set scheduling_algorithm_provider = "" -%}
+{% if grains.scheduling_algorithm_provider is defined -%}
+ {% set scheduling_algorithm_provider = "--algorithm-provider=" + grains.scheduling_algorithm_provider -%}
+{$ endif -%}
+
+{% set params = params + log_level + " " + feature_gates + " " + scheduling_algorithm_provider -%}
```

Comment:
Discussed offline.
I initially though that there won't be any reason to switch just this scheduling and leave cluster autoscaler on, but it may not necessary be true.
So it may make sense to leave those variables.